taichi/transforms/whole_kernel_cse.cpp

```diff
@@ -28,8 +28,6 @@ class WholeKernelCSE : public BasicStmtVisitor {
     // Move common statements at the beginning or the end of both branches
     // outside.
     if (if_stmt->true_statements && if_stmt->false_statements) {
-      auto block = if_stmt->parent;
-      int current_stmt_id = block->locate(if_stmt);
```

Comment:
Can I remove these two lines? They are causing a warning (unused variables).

Reply:
Oh sure. Sorry for not removing all debug statements...